sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala

```diff
@@ -1047,7 +1047,8 @@ trait ArraySortLike extends ExpectsInputTypes {
         s"int $c = ${ctx.genComp(elementType, s"(($jt) $o1)", s"(($jt) $o2)")};"
       }
       val nonNullPrimitiveAscendingSort =
```

Comment:
`nonNullPrimitiveNumericAscendingSort` or `fastSortCode` preferred?

Reply:
@maropu Lets keep the variable name the same .. but add a private def like you suggest in the following comment.